src/sqlfluff/rules/L026.py

```diff
@@ -66,7 +66,7 @@ def _eval(self, context: RuleContext) -> EvalResultType:
         self.force_enable: bool
 
         if (
-            context.dialect.name in ["bigquery", "hive", "redshift", "sparksql"]
+            context.dialect.name in ["bigquery", "hive", "redshift", "soql", "sparksql"]
```

Comment:
So you think we should move these to a class variable (or whatever it’s called), similar to how we’ve done for dialect exceptions in other rules? Because it’s almost like a bit of config like those are.